changelogs/unreleased/gh-7226-admin-grant-error.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/box
+
+* Fix internal error on granting rights to admin user. Now granting still
+fail but for proper reason and proper error message (right is already granted).
+gh-7226.
```

Comment:
We usually use for following format of changelog: `Fixed/changed/done whatever (gh-XXXX).`